src/components/content-tab-assistant.tsx

```diff
@@ -212,11 +225,27 @@ export function ContentTabAssistant( { selectedSite }: ContentTabAssistantProps
 				data-testid="assistant-chat"
 				className={ cx(
 					'flex-1 overflow-y-auto p-8 flex flex-col-reverse',
-					! isAuthenticated && 'flex items-start'
+					isOffline && ! isAuthenticated
+						? 'flex items-center justify-center'
+						: ! isAuthenticated
+						? 'flex items-start'
+						: ''
```

Comment:
We can consider "simplify"  the logic. Some alternatives that could work:

```
!isAuthenticated
    ? isOffline
        ? 'flex items-center justify-center'
        : 'flex items-start'
    : ''
```
or 

```
!isAuthenticated && isOffline && 'flex items-center justify-center',
!isAuthenticated && !isOffline && 'flex items-start'

Reply:
Refactored in https://github.com/Automattic/studio/pull/271/commits/98805fda36a47f00a904a3116633563951c20576